src/pvecontrol/config_default.yaml

```diff
@@ -4,10 +4,14 @@ clusters:
   - name: cluster1
     host: 127.0.0.1
     user: pvecontrol@pve
+    timeout: 5
```

Comment:
This default timeout is not big enough. On big storage listing requests this can be normal that it takes up to 60 seconds to respond.